docs/source/docker_setup.rst

```diff
@@ -1,9 +1,15 @@
-Dallinger with Docker
-=====================
+Image generation
+================
 
-With the release of Dallinger version 5.0.0, we have created a Python script that uses `docker-compose <https://docs.docker.com/compose/>`__ to provide an automated installation and configuration of Dallinger to run experiments.
+Starting from version XXX, Dallinger can generate docker images to be used for deployment.
 
-The code and detailed instructions can be found in this `github repository <https://github.com/Dallinger/Dockerfiles/blob/master/README.md/>`__.
+A base image is provided in the docker registry: `dallinger/dallinger:<version>`
+for a specific version, and `dallinger/dallinger:<version>` for the latest one.
 
-Please note that we consider this to be a working yet experimental method of running Dallinger. It adds an extra level of complexity which can potentially get in the way when trying to create and debug a new experiment as debugging is more diffcult than when using Dallinger natively or in a virtual machine.
-Having said that, there are can be certain advantages to this method, since Docker can install everything required to run Dallinger quickly in comparison to installing all the requirements yourself, and on platforms such as Microsoft Windows where a native installation is not possible.
\ No newline at end of file
+The dallinger CLI can generate an image for an experiment and upload it to heroku.
```

Comment:
This definitely needs some more install and usage documentation.

Reply:
When I tried to run `dallinger docker debug` from `demos/dlgr/demos/bartlett1932` the `docker-compose` command failed during the `Dockerfile.worker` build with:
```
 => [2/4] COPY . /experiment                                               0.1s
 => [3/4] WORKDIR /experiment                                              0.0s
 => ERROR [4/4] RUN bash -c "python3 -m pip install -r <(grep -v ^dalling  0.4s
------
 > [4/4] RUN bash -c "python3 -m pip install -r <(grep -v ^dallinger requirements.txt)":
#8 0.375 /usr/bin/python3: No module named pip
------
executor failed running [/bin/sh -c bash -c "python3 -m pip install -r <(grep -v ^dallinger requirements.txt)"]: exit code: 1
ERROR: Service 'worker' failed to build
```
Not sure how that would have happened since `python3-pip` should be in the runtime image.

Reply:
Thanks! I had generated updated base images locally using `docker build`, but it's not clear to me how I would make the `dallinger docker debug` command use them. Maybe something to add to the documentation? There should probably be a clear way to build updated local base images and ensure that the `dallinger` command uses them.

Reply:
@alecpm The images used are hardcoded. Now they are `dallingerimages/dallinger` and `dallingerimages/dallinger-bot`.
If you don't have them locally docker will try and download them from the public registry.

I'm very surprised you get that error. That's why my first thought was of you having the wrong image.
The image definitely ships `pip` for python 3.
Can you try to run `docker run --rm dallingerimages/dallinger python3 -m pip install nltk`?
For me it succeeds, but given your error above I expect it to fail on your machine.
Is this the sha of the image you're using?
```
docker inspect dallingerimages/dallinger|grep Id
        "Id": "sha256:4ef4b3db35e0d0a00495d95ced04fce16ba67757ba54944c690f283caabac5f0",
```

Reply:
@silviot This is probably all be down to me being a total Docker newbie. After running `docker build .` in the Dallinger dir (which does seem to build all the images), I end up with a single orphaned image:
```
% docker image ls 
REPOSITORY               TAG       IMAGE ID       CREATED          SIZE
<none>                   <none>    acb1300c29d5   39 seconds ago   2.5GB
```
Perhaps there's a build command I should be using to ensure that the build gets assigned to the correct repo?

Reply:
@alecpm I included the needed commands in the docs now. Your build command was missing the tag to assign to the built image (passed with `-t`) and the target to build (our Dockerfile provides more than one target, and by default I believe docker will build the last one in the file, so the chromedriver image in this case) that you can specify with `--target`.